src/client/eppo-client.ts

```diff
@@ -459,4 +516,16 @@ export default class EppoClient implements IEppoClient {
     const shard = getShard(`exposure-${subjectKey}-${flagKey}`, subjectShards);
     return shard <= percentExposure * subjectShards;
   }
+
+  private isInHoldoutSample(
+    subjectKey: string,
+    flagKey: string,
+    experimentConfig: IExperimentConfiguration,
+    holdout: IHoldout,
+  ): boolean {
+    const { subjectShards } = experimentConfig;
+    const { percentExposure } = holdout;
+    const shard = getShard(`holdout-exposure-${subjectKey}-${flagKey}`, subjectShards);
```

Comment:
Should not shard by flagKey because the subjects should be assigned to the same holdout irrespective of which flag they're in. Aaron called it the "universal control" or we can also think of it as a "global" holdout which is independent of which flag it's on